pkg/syfs/syfs.go

```diff
@@ -43,20 +43,26 @@ func ConfigDir() string {
 }
 
 func configDir() string {
-	user, err := user.Current()
-	if err != nil {
-		sylog.Warningf("Could not lookup the current user's information: %s", err)
+	configDir := os.Getenv("SINGULARITY_CONFIGDIR")
+	if configDir == "" {
+		sylog.Warningf("Environment variable SINGULARITY_CONFIGDIR is not set")
```

Comment:
This shouldn't be a warning. Most users will not set the environment variable, so we don't want a warning every time singularity is run by them.

Reply:
So it would be `Infof`?

Reply:
No, please remove it. We don't want any output when singularity is using the default location.

Reply:
Ok